src/app/app-routing.module.ts

```diff
@@ -1,10 +1,17 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { LoginComponent } from './login/login.component';
+import { RegisterComponent } from './register/register.component';
+import { ErrorComponent } from './error/error.component';
 
-const routes: Routes = [];
+const routes: Routes = [
```

Comment:
please add redirect of the default page to Login page(because now when you open http://localhost:4200/ you will always be shown 404 page)
example: 
  { path: '', redirectTo: 'login', pathMatch: 'full'}